src/Process_Common.cc

```diff
@@ -940,22 +944,10 @@ void Process_Common_NanoAOD()
         if (W_MD > fjWPloose_MD) WPid_MD = 1; 
         if (W_MD > fjWPmedium_MD) WPid_MD = 2; 
         if (W_MD > fjWPtight_MD) WPid_MD = 3; 
-        if (fatjet_msoftdrop >= 65. and fatjet_msoftdrop <= 105. and fatjet_p4.pt() > 200.)
-        {
-            ana.tx.pushbackToBranch<int>("Common_fatjet_WP", WPid);
-            ana.tx.pushbackToBranch<int>("Common_fatjet_WP_MD", WPid_MD);
-            ana.tx.pushbackToBranch<int>("Common_fatjet_WP_antimasscut", -999);
-            ana.tx.pushbackToBranch<int>("Common_fatjet_WP_MD_antimasscut", -999);
-        }
-        else
-        {
-            ana.tx.pushbackToBranch<int>("Common_fatjet_WP", -999);
-            ana.tx.pushbackToBranch<int>("Common_fatjet_WP_MD", -999);
-            ana.tx.pushbackToBranch<int>("Common_fatjet_WP_antimasscut", WPid); // store W DNN cut even off mass peak
-            ana.tx.pushbackToBranch<int>("Common_fatjet_WP_MD_antimasscut", WPid_MD); 
-            WPid = -999;                                                       // I reset WPid to not store the fatjet SF for offmass peak
-            WPid_MD = -999;
-        }
+        ana.tx.pushbackToBranch<int>("Common_fatjet_WP", WPid);
+        ana.tx.pushbackToBranch<int>("Common_fatjet_WP_MD", WPid_MD);
+        ana.tx.pushbackToBranch<int>("Common_fatjet_WP_antimasscut", -999);
```

Comment:
These antimass cut branches are no longer used, can they be removed? This should also be propagated to the VVVtree portion of the code (L1629)

Reply:
I listed them out in my slides today. 
1. For anti-mass-cut branch, it is a "place-holder" for now to prevent crushes and will be removed once we make sure no one is using it
2. We need to do VVVtree change, though I'm not familiar with it, we can integrate that in next PR